specification/purview/data-plane/Azure.Analytics.Purview.DevopsPolicies/preview/2022-11-01-preview/purviewDevopsPolicy.json

```diff
@@ -0,0 +1,611 @@
+          }
+        },
+        "x-ms-examples": {
+          "DevopsSqlPerfMonitorPolicy_Get": {
+            "$ref": "./examples/DevopsSqlPerfMonitorPolicy_Get.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "DevopsPolicyDecisionRule": {
+      "type": "object",
+      "required": [
+        "effect",
+        "principals",
+        "principalGroups"
+      ],
+      "properties": {
+        "effect": {
+          "description": "The effect for rule",
+          "enum": [
+            "Permit"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "Decision",
+            "modelAsString": true
+          }
+        },
+        "purviewRoleName": {
+          "type": "string",
+          "description": "This is field will indicate the role of a devops policy."
+        },
+        "principals": {
+          "type": "array",
+          "items": {
+            "type": "string",
+            "description": "Azure Active Directory Ids for users"
+          }
+        },
+        "principalGroups": {
+          "type": "array",
+          "items": {
+            "type": "string",
+            "description": "Azure Active Directory Ids for groups"
+          }
+        },
+        "resourceAzurePath": {
+          "type": "string",
+          "description": "Optional resource path of the azure resource"
+        }
+      }
+    },
+    "DevopsPolicyScope": {
+      "type": "object",
+      "required": [
+        "type",
+        "referenceName"
+      ],
+      "properties": {
+        "type": {
+          "description": "The type of reference",
+          "default": "DataSourceReference",
+          "type": "string"
+        },
+        "referenceName": {
+          "description": "The name of resource as registered on the purview portal",
+          "type": "string"
+        }
+      }
+    },
+    "ErrorModel": {
+      "description": "The error model for policy",
+      "required": [
+        "code",
+        "message"
+      ],
+      "type": "object",
+      "properties": {
+        "code": {
+          "description": "The error code",
+          "minLength": 1,
+          "type": "string"
+        },
+        "message": {
+          "description": "The error message",
+          "minLength": 1,
+          "type": "string"
+        },
+        "details": {
+          "description": "The error details",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ErrorModel"
+          }
+        }
+      }
+    },
+    "ErrorResponseModel": {
+      "description": "The error response model for policy",
+      "required": [
+        "error"
+      ],
+      "type": "object",
+      "properties": {
+        "error": {
+          "$ref": "#/definitions/ErrorModel"
+        }
+      }
+    },
+    "DevopsPolicy": {
+      "required": [
+        "decisionRules",
+        "scope"
+      ],
+      "type": "object",
+      "properties": {
+        "policyName": {
```

Comment:
Where does "policyName" come from and why is it here? There is no "policyName" parameter on create or on any other operation for that matter.

Reply:
Removed the policyName property.